platform/srf06-cc26xx/contiki-conf.h

```diff
@@ -78,7 +78,12 @@
 
 /* Configure ContikiMAC for when it's selected */
 #define CONTIKIMAC_CONF_WITH_CONTIKIMAC_HEADER  0
-#define CONTIKIMAC_CONF_WITH_PHASE_OPTIMIZATION 0
+/* Phase optimization: enable, but increase the guard and max strobe time 2 times */
+#ifndef CONTIKIMAC_CONF_WITH_PHASE_OPTIMIZATION
+#define CONTIKIMAC_CONF_WITH_PHASE_OPTIMIZATION 1
```

Comment:
Are you 100% certain that enabling phase optimisation won't lead to a barrage of issues? :) I'm willing to take your word for it here.